src/components/OrdersWidget/index.tsx

```diff
@@ -99,6 +99,9 @@ const OrdersWidget: React.FC = () => {
         // reset selections
         setOrders(orders.filter(order => !markedForDeletion.has(order.id)))
         setMarkedForDeletion(new Set<string>())
+
+        // update the list of orders
+        forceOrdersRefresh()
```

Comment:
do we need this if we subscribe instead to userAccount changes via subs?

Reply:
or networkId or whatever or block

Reply:
or wait is this what we talked about on the last call that we don't want to do that, don't remember now :thinking:

Reply:
this is to refetch everything on order deletion, regardless network or account or whatever

Reply:
All of the above.